appdatabase/database.go

```diff
@@ -107,3 +115,74 @@ func GetDBFilename(db *sql.DB) (string, error) {
 
 	return "", errors.New("no main database found")
 }
+
+func MigrateEnsUsernames(db *sql.DB) error {
+
+	// 1. Check if ens_usernames table already exist
+
+	row := db.QueryRow("SELECT exists(SELECT name FROM sqlite_master WHERE type='table' AND name='ens_usernames')")
+	tableExists := false
+	err := row.Scan(&tableExists)
+
+	if err != nil && err != sql.ErrNoRows {
+		return err
+	}
+
+	if tableExists {
+		return nil
+	}
+
+	// -- 1. Create new ens_usernames table
```

Comment:
could we have the structure migration in a normal migration file and only the data migration in go?